internal/service/organizations/account.go

```diff
@@ -458,10 +458,13 @@ func statusAccountStatus(ctx context.Context, conn *organizations.Organizations,
 	}
 }
 
-func waitAccountDeleted(ctx context.Context, conn *organizations.Organizations, id string) (*organizations.Account, error) {
+func waitAccountSuspended(ctx context.Context, conn *organizations.Organizations, id string) (*organizations.Account, error) {
 	stateConf := &resource.StateChangeConf{
-		Pending:      []string{organizations.AccountStatusPendingClosure},
-		Target:       []string{},
+		Pending: []string{organizations.AccountStatusPendingClosure, organizations.AccountStatusActive},
+		// From the time you close the account until 90 days expire, closed accounts are visible in
+		// your organization with the SUSPENDED state.
+		// Ref: https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_accounts_close.html
+		Target:       []string{organizations.AccountStatusSuspended},
 		Refresh:      statusAccountStatus(ctx, conn, id),
```

Comment:
Ah, thanks! I'll revert this commit.

I think`AccountStatusActive` may still need added to the `Pending` field slice to account for the delay before active accounts return a suspended status (and get handled as `NotFound` by the finder function). I suspect this might have been the actual cause for the waiter failures I was observing while testing, though I don't have the output around anymore to confirm.

Reply:
👍